src/config/index.ts

```diff
@@ -34,6 +36,14 @@ const packageFile = JSON5.parse(
   fs.readFileSync(path.join(projectRoot, 'package.json'), 'utf8')
 ) as PackageFile
 
+const result = schema.safeParse(configFile)
```

Comment:
You don't need the `interface ConfigFile` anymore. If you need the type you can infer it from Zod schema: `type ConfigFile = z.infer<typeof schema>`